prime-angular-frontend/src/app/shared/components/forms/email-form/email-form.component.html

```diff
@@ -8,6 +8,7 @@
           <input matInput
                  placeholder="Email"
                  formControlName="email"
+                 (keydown.space)="$event.preventDefault()"
```

Comment:
This solution doesn't work for emails copied and pasted from somewhere else  😒

Reply:
Created a directive to trim the space. It should work better.